index.d.ts

```diff
@@ -0,0 +1,14 @@
+declare module 'react-media' {
```

Comment:
This line is unnecessary when you bundle typings with the package like we're doing here. The module name will be the one in `package.json`.